pubspec.yaml

```diff
@@ -39,8 +41,12 @@ dev_dependencies:
   # Not yet use solid-lints. It requires analyze 6.2.0 and
   # dart-code-metrics fails.
   # solid_lints: ^0.1.1
+  riverpod_generator: ^2.4.0
```

Comment:
Do we use `riverpod_generator` anywhere?

Reply:
removing now